src/ert/config/ert_config.py

```diff
@@ -608,6 +608,11 @@ def _create_list_of_forward_model_steps_to_run(
                             context=fm_step.name,
                         ),
                     )
+                except Exception as e:  # type: ignore
+                    ConfigWarning.warn(str(e), context=fm_step.name)
```

Comment:
I think both the log and the warning should have the text `Unexpected exception in forward model validation of {fm_step.name}: ` as all `ConfigWarning`s are logged in the `Suggestor` we don't need an additional log statement.